modules/110-istio/openapi/config-values.yaml

```diff
@@ -207,6 +207,20 @@ properties:
         - {}
         - jaegerURLForUsers: https://tracing-service:4443/jaeger
           jaegerGRPCEndpoint: http://tracing.myjaeger.svc:16685/
+  dataPlane:
```

Comment:
Why is this flag required in the first place? What will happen if I change it, e.g., should I expect downtime?

Reply:
Get rid of empty dataPlane section pls.

Reply:
It is still here.

Reply:
It is still here.